src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/probation/ProbationSearchResult.kt

```diff
@@ -1,8 +1,16 @@
 package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.probation
 
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceType
+import java.time.LocalDate
+
 data class ProbationSearchResult(
   val name: Name,
   val identifiers: Identifiers,
   val manager: Manager,
   val allocationDate: String,
+  val licenceType: LicenceType? = null,
+  val licenceStatus: LicenceStatus? = null,
+  val releaseDate: LocalDate? = null,
+  val isOnProbation: Boolean? = null,
```

Comment:
I'd keep this class "clean" and just match what is returned from the API rather than have the enriched fields on here as a mechanism to pass the data across on the DTO. 


Think it'd be better to do the transform in the mapForNull and pass the licence to `transformToModelProbationResult` as a parameter?

Reply:
Done